tasmota/i18n.h

```diff
@@ -122,6 +122,11 @@
 #define D_JSON_MODEL "Model"
 #define D_JSON_MOISTURE "Moisture"
 #define D_JSON_MQTT_COUNT "MqttCount"
+// xsns_96_QMC5883.ino
+#define D_JSON_MX "Compass  X-Axis"
+#define D_JSON_MY "Compass  Y-Axis"
+#define D_JSON_MZ "Compass Z-Axis"
+#define D_JSON_HEADING "Compass Heading"
```

Comment:
These are wrong names for JSON keys
A JSON key must be a valid variable name in Javascript with only alphanumeric characters  + underscore and starting with an alpha character.
Spaces, dash or other separators are illegal.
Please fixe